examples/dapperish/dapper.go

```diff
@@ -1,11 +1,11 @@
 package dapperish
 
 import (
+	"github.com/opentracing/basictracer-go"
```

Comment:
can we move daperish into basic tracer too?  these recursive imports look strange.